src/app/Login/page.module.css

```diff
@@ -0,0 +1,56 @@
+.EmailContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 80px;
+}
+.PasswordContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 140px;
+}
+.label {
+  margin-bottom: 10px;
+  display: flex;
+}
+.span {
+  margin-top: 20px;
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+}
+
+.welcomeBackcontainer {
+  margin: 100px;
+  text-align: center;
+  font-size: 50px;
+  text-align: center;
+  color: #ffffff;
```

Comment:
delete, we dont want to hardcode colors on the page or component level otherwise it will make light vs dark mode buggy down the line. text colors should only be defined in `globals.css`, you don't have to worry about this since that's a different story